RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -1530,6 +407,7 @@ class DeepTauId : public deep_tau::DeepTauBase {
           if (!(pred >= 0 && pred <= 1))
             throw cms::Exception("DeepTauId")
                 << "invalid prediction = " << pred << " for tau_index = " << tau_index << ", pred_index = " << k;
+          std::cout << "tau_index " << tau_index << " k " << k << " pred " << pred << std::endl;
```

Comment:
cout is not allowed, use MessageLogger (or protect with debug flag)

Reply:
removed. (this is a remnant during debugging. forgot to clean. sorry.)